src/appShell/App/App.tsx

```diff
@@ -36,23 +40,21 @@ class App extends React.Component {
 
   get devTools () {
     if (__DEBUG__) {
-      if (!window.devToolsExtension) {
-        const DevTools = require('../DevTools/DevTools').default
-        return <DevTools />
+      if (!(window as any).devToolsExtension) {
+        const DevTools = require('../DevTools/DevTools').default;
+        return <DevTools />;
       }
     }
   }
 
   render () {
      return (
        <Provider store={this.props.store}>
-         <div style={{ height: '100%' }}>
+         <div style={{ height: 700 }}>
```

Comment:
Yeah, I think this component itself should accept an optional fixed height prop, which would be put on its outermost el.